fdbmonitor/fdbmonitor_lib.cpp

```diff
@@ -0,0 +1,923 @@
+/*
+ * fdbmonitor_lib.cpp
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2022 Apple Inc. and the FoundationDB project authors
```

Comment:
Update year?

Reply:
I kept it this way because commonly updated files like fdbserver.actor.cpp have the same end year. I can send a separate PR to update all instances.

Reply:
11553